### PR TITLE
Alerting docs: backport to 10

### PR DESCRIPTION
**Pull request by brendamuir**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by brendamuir** [comment]
@jdbaldry this is all ok except for the create a mimir or loki alerting rule. there i don't want those new green changes. i checked back in the topic and can't see how to update it...can you help?

**Review by jdbaldry — COMMENTED** [review]
It looks like the backport has included a lot more changes than desired

Let me know if you want me to double check each of the additional changes. It might be a lot of me asking "is this intended?" or "is this correct in this version?".

Alternatively, we can ditch the backport PR and look to manually implement the changes specifically for this branch which might be easier.

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 1]
```diff
@@ -57,14 +57,20 @@ To create a Grafana Mimir or Loki managed recording rule
...
-1. Click **Save** to save the rule or **Save and exit** to save the rule and go back to the Alerting page.
+1. Click **Save rule** to save the rule or **Save rule and exit** to save the rule and go back to the Alerting page.
+
+{{% docs/reference %}}
+[annotation-label]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/fundamentals/annotation-label"
+[annotation-label]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/annotation-label"
+
+[configure-grafana]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana"
+[configure-grafana]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana"
+{{% /docs/reference %}}
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:76]:
```suggestion
```

To me, it doesn't look like any of these are used in this file so it is best to remove them.

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 2]
```diff
@@ -63,4 +65,89 @@ If you do not want to manage alerting rules for a particular Loki or Prometheus
 1. In Step 5, add custom labels.
    - Add custom labels selecting existing key-value pairs from the drop down, or add new labels by entering the new key or value .
 1. Click **Save** to save the rule or **Save and exit** to save the rule and go back to the Alerting page.
-1. Next, create a notification for the rule.
+1. # Next, create a notification for the rule.
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:68]:
```suggestion
1. Next, create a notification for the rule.
```

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 3]
```diff
@@ -63,4 +65,89 @@ If you do not want to manage alerting rules for a particular Loki or Prometheus
...
+
+{{% docs/reference %}}
+[alerting]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting"
+[alerting]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting"
+
+[annotation-label]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/fundamentals/annotation-label"
+[annotation-label]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/annotation-label"
+{{% /docs/reference %}}
+
+> > > > > > > 23a873b33f (Alerting docs: fixes next and latest link errors (#75437))
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:153]:
```suggestion
```

I'm guessing these changes aren't expected.

File: docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md [hunk 4]
```diff
@@ -137,7 +137,9 @@ deleteRules:
 
 Create or delete contact points in your Grafana instance(s).
 
-1. Create a YAML or JSON configuration file.
+1. Create a contact point in Grafana.
+1. Use the [Alerting provisioning API][alerting_provisioning] export endpoints to download a provisioning file for your contact point.
```
Comment (jdbaldry) [inline docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md:141]: There's no `docs/reference` section at the bottom of this page so this link won't work.

**Pull request closed without merging**